tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
-envlist = py27, py35, pep8, docs
+envlist = py27, py36, pep8, docs
```

Comment:
I think we should still keep and test compatibility for python versions < 3.6.

Reply:
I also suggest adding `py36` instead of replacing `py35` here. right now https://github.com/figo-connect/schwifty/pull/11/files#diff-b91f3d5bd63fcd17221b267e851608e8R17 is doing nothing and doctests are not executed anymore for python3.